{
    "issue": {
        "title": "Make merge_arrow_pr script work with Python 3",
        "body": "***Note**: This issue was originally created as [ARROW-339](https://issues.apache.org/jira/browse/ARROW-339). Please see the [migration documentation](https://gist.github.com/toddfarmer/12aa88361532d21902818a6044fda4c3) for further details.*\n\n### Original Issue Description:\nSee the patches made to https://github.com/pandas-dev/pandas/blob/master/scripts/merge-py.py",
        "created_at": "2016-10-17T13:13:36.000Z",
        "updated_at": "2019-06-03T13:18:18.000Z",
        "labels": [
            "Migrated from Jira",
            "Component: Developer Tools",
            "Type: bug"
        ],
        "closed": true,
        "closed_at": "2016-10-29T08:02:43.000Z"
    },
    "comments": [
        {
            "created_at": "2016-10-29T08:02:43.848Z",
            "body": "***Note**: [Comment](https://issues.apache.org/jira/browse/ARROW-339?focusedCommentId=15617690) by Uwe Korn (uwe):*\nIssue resolved by pull request 188\n<https://github.com/apache/arrow/pull/188>"
        }
    ]
}